front/components/data_source_view/DataSourceViewSelector.tsx

```diff
@@ -315,6 +316,7 @@ export function DataSourceViewSelector({
   isRootSelectable,
   defaultCollapsed = true,
 }: DataSourceViewSelectorProps) {
+  const [isChecked, setIsChecked] = useState<boolean | "partial">(false);
```

Comment:
I don't understand the reason of duplicating the state here, the source of truth should only be the selectionConfiguration prop

Reply:
You right, got rid of this unnecessary state 👍🏼